test/Parse/recovery.swift

```diff
@@ -44,15 +44,15 @@ func test(a: BadAttributes) -> () { // expected-note * {{did you mean 'test'?}}
 //===--- Recovery for braced blocks.
 
 func braceStmt1() {
-  { braceStmt1(); } // expected-error {{braced block of statements is an unused closure}} expected-error {{expression resolves to an unused function}}
+  { braceStmt1(); } // expected-error {{closure expression is unused}} expected-note {{use 'do' keyword to designate a separate block of statements}} {{3-3=do }}
```

Comment:
I was also thinking of using the word `scope` or `scoped` somehow.

But I think `did you mean to use a 'do' statement?` could work... (especially since "`do` statement" is the terminology used in [the documentation](https://developer.apple.com/library/content/documentation/Swift/Conceptual/Swift_Programming_Language/Statements.html#//apple_ref/swift/grammar/do-statement))